### DISCUSSION TITLE
sidebar/sanfene/rocketmq

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#258] [Announcements]
# sidebar/sanfene/rocketmq

下载次数超 1 万次，1.1 万字 45 张手绘图，详解 23 道 RocketMQ 面试高频题（让天下没有难背的八股），面渣背会这些 RocketMQ 八股文，这次吊打面试官，我觉得稳了（手动 dog）。

https://javabetter.cn/sidebar/sanfene/rocketmq.html

**Comment by Lweiis** [comment]
点个赞，刷了好几篇了，很全面也很形象，感谢分享

**Comment by Kddk-K** [comment]
9.14一刷
